musicbot/bot.py

```diff
@@ -183,6 +186,19 @@ def _delete_old_audiocache(self, path=AUDIO_CACHE_PATH):
 
         return True
 
+    def _setup_aliases(self):
+        if len(self.aliases) > 0:
+            for command_name, alias in self.aliases.items():
+                if hasattr(self, 'cmd_{}'.format(alias)):
+                    log.debug('Command named {} already exists!'.format(alias))
```

Comment:
Let's be more descriptive with this, and change it to a warning instead - people should be aware if there is an issue with their aliases, rather than it being hidden away in the log file.

Reply:
Now raises `HelpfulError`s to help the end user